### DISCUSSION TITLE
Layout issue with responsive SVG

### DISCUSSION THREAD

**Discussion by ledermann** [tailwindlabs/tailwindcss#5582] [Help]
Hello everyone, I have a problem with the responsiveness of a layout using inline SVGs. There is a minimal example here:
https://play.tailwindcss.com/aVCqSSu60T?size=800x400

I want to center the SVGs horizontally without any gap between them. Is this possible?

**Comment by vaibhavshn** [comment]
Centered them with `flex justify-center`: https://play.tailwindcss.com/VIXLeqjpvH?size=800x400

If you do not want gaps however, you'll have to provide a width to the svg's, right now the 3 svgs are filling up the parent by using 1/3rd of the width with `flex-1`.

**Comment by ledermann** [comment]
Yes, the `flex justify-center` was missing. But the most difficult art is the gap between the SVGs. I have solved the issue by increasing the horizontal padding of the outer `div` via JavaScript based on the current container width and the number of SVGs. Ugly, but works.

I'm curious what exactly do you mean with "providing a width to the SVG". Adding a `width` attribute to the SVG does not change the positioning for me, it just reduces or increases the SVG size.

**Comment by simonswiss** [comment]
Hey!

Not sure if this is what you want, but here's a Play demo where I have switched from using _borders_ to using `ring` utilities (which are implemented with `box-shadow`:

https://play.tailwindcss.com/kQ2DA4oHh0?size=632x677

I also added a `z-10` on the `peer-checked` element, so that the `ring` always shows on top of the other circles.

Not sure if this is the desired effect, but it sounds much easier to maintain than calculating the padding to compensate for the transparent border in your version.

---

### Alternative option

Another option would be to only give the circles a border on `peer-checked`, so get rid of the `border-8 border-transparent` and replace with `peer-checked:border-8 peer-checked:border-blue-500`

Hope it helps!

**Comment by ledermann** [comment]
@simonswiss Thank you very much for your input. I like using `ring` instead of `border`, but the main issue is something different, and it's caused by the SVGs. I want the dots to be horizontally centered with the **same small gap** between them. And this should be fully responsive, so if there is more (or less) room, the dots must grow (or shrink).

Let me visualize this:

| Currently        | What I want | 
| ------------- |-------------| 
| <img width="702" alt="currently" src="https://user-images.githubusercontent.com/3557/135224645-b0cba411-383c-45ca-b7eb-666c3d07faa6.png">      | <img width="701" alt="prefered" src="https://user-images.githubusercontent.com/3557/135224637-2a18eb97-4842-4e6a-8854-ba8fc840867c.png"> | 

BTW: The dots are just placeholders for "real" SVGs.
